Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -54,14 +54,10 @@ export const PublishProfileDialog: React.FC<PublishProfileDialogProps> = (props)
   const [page, setPage] = useState(Page.ProfileForm);
   const [publishSurfaceStyles, setStyles] = useState(defaultPublishSurface);
   const { provisionToTarget, addNotification } = useRecoilValue(dispatcherState);
-
-  const [dialogTitle, setTitle] = useState({
-    title: current ? formatMessage('Edit publishing profile') : formatMessage('Add new publishing profile'),
-    subText: formatMessage('A publishing profile provides the secure connectivity required to publish your bot. '),
-  });
-
   const [selectedType, setSelectType] = useState<PublishType | undefined>();
 
+  const [dialogTitle, setTitle] = useState(formatDialogTitle());
```

Comment:
As we discussed, the formatDialogTitle needs to be moved up so it is declared before it is used.

Reply:
done